resources/test_data/tbl/int_sorted_only_null.tbl

```diff
@@ -0,0 +1,7 @@
+a
```

Comment:
What is "sorted" about this file?

Reply:
Nothing and everything. I think this philosophical question has to be discussed offline during a Gin soiree. Otherwise this PR cannot be merged.

Reply:
This sounds like you already started drinking?